README.md

```diff
@@ -134,6 +140,29 @@ pre-trained model checkpoints.
 For technical documentation, please see the [NeMo Framework User
 Guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/playbooks/index.html).
 
+## What's New in NeMo 2.0
+
+NVIDIA NeMo 2.0 introduces several significant improvements over its predecessor, NeMo 1.0, enhancing flexibility, performance, and scalability.
+
+- **Python-Based Configuration** - NeMo 2.0 transitions from YAML files to a Python-based configuration, providing more flexibility and control. This shift makes it easier to extend and customize configurations programmatically.
+
+- **Modular Abstractions** - By adopting PyTorch Lightning’s modular abstractions, NeMo 2.0 simplifies adaptation and experimentation. This modular approach allows developers to more easily modify and experiment with different components of their models.
+
+- **Scalability** - NeMo 2.0 seamlessly scaling large-scale experiments across thousands of GPUs using [NeMo-Run](https://github.com/NVIDIA/NeMo-Run).
+
+- **Training Throughput** - NeMo 2.0 efficiently handles large-scale models and datasets. It manages large batch sizes and sequence lengths, distributing model components across GPUs through model parallelization.
+
+Overall, these enhancements make NeMo 2.0 a powerful, scalable, and user-friendly framework for AI model development.
+
+> [!IMPORTANT]  
+> NeMo 2.0 is currently supported by the LLM and VLM collections.
+
+### Get Started with NeMo 2.0
+
+- For more information about NeMo 2.0, see the [NeMo Framework User Guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo-2.0/index.html).
```

Comment:
Recommend linking directly to quick start assets as well: https://docs.nvidia.com/nemo-framework/user-guide/latest/nemo-2.0/quickstart.html

Reply:
Infact, Quickstart should be the very first bullet. More information, migration etc. should be after that.

Reply:
Second bullet could be the recipes folder we have: 
"[NeMo 2.0 Recipes](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/llm/recipes) contains additional examples of launching large-scale runs using NeMo 2.0 and NeMo-Run."